finalfate/gameInput.js

```diff
@@ -126,6 +128,21 @@ function getKeyPress(event) {
     }
 }
 
+function getMouseDown(event) {
+    if (typeof event === 'object') {
+    //keycode 0 -> most browsers
+    //keycode 1 -> internet explorer left click -> middle mouse on other browsers
+      switch (event.button) {
+        case 0:
+          shoot = 5;
+          break;
+        case 1:
+          shoot = 5;
+          break;
+      }
```

Comment:
The switch statement (also the one from lines 271 to 281) could have been simplified, but do not worry about this.